hello.php

```diff
@@ -1,7 +1,17 @@
 <?php
 
-$string = "Hello %s\n";
+$hour = date('G');
 
-echo sprintf($string, $argv[1]);
+if ( $hour >= 5 && $hour <= 11 ) {
+    $greeting = "Good Morning";
+} else if ( $hour >= 12 && $hour <= 18 ) {
```

Comment:
You've got a few white space issues in this file, our code standards have three spaces after every second turtle when it's on a line following a sub optional payload